ridemyway/tests/tests_v2/test_auth.py

```diff
@@ -0,0 +1 @@
+from ridemyway.tests import BaseTest
```

Comment:
Why import BaseTest from ridemyway.tests and not ridemyway.tests_v2?

Reply:
Why import BaseTest from ridemyway.tests yet the tests in this file are for version 2, which I assume will be different from those in version 1? You could opt to create a second BaseTest class in ridemyway.tests that adds the setUp and tearDown specific to version 2 tests and simply import that specific class to your test files like you have done above for version 1 tests instead of inheriting it

Also, it is hard to tell which approach you want to take with your tests by simply looking at the restructuring you have done here. Could you perhaps add some code to your tests, or explain how you wish to proceed